gpt4all-chat/src/chatllm.h

```diff
@@ -28,12 +28,60 @@ using namespace Qt::Literals::StringLiterals;
 class QDataStream;
 
 // NOTE: values serialized to disk, do not change or reuse
-enum LLModelType {
-    GPTJ_  = 0, // no longer used
-    LLAMA_ = 1,
-    API_   = 2,
-    BERT_  = 3, // no longer used
+enum class LLModelTypeV0 { // chat versions 2-5
```

Comment:
I haven't gone back and looked but I'll just assume you have compiled this correctly from older versions and/or commit history of older versions.

Reply:
Yes, this comes from commit 4219c0e2.